### DISCUSSION TITLE
Error for UI testing using UNO.UItest

### DISCUSSION THREAD

**Discussion by Chinmay062001** [unoplatform/uno#18821] [General]
Used default setup, No changes done still getting following error
![image](https://github.com/user-attachments/assets/42e425e2-6bc8-46cb-9a88-628bc80e7b8e)

**Comment by Chinmay062001** [comment]
@jeromelaban could you look and suggest correction? If possible could u also share a setup without any errors

**Reply by jeromelaban** [reply]
This is an unknown error, not sure where the proxy error comes from. Does it happen when running this test manually?

**Reply by Chinmay062001** [reply]
Heyy @jeromelaban - it is a new issue. I just created a blank project and try running test cases. This error pooped after building and testing the app.

**Reply by jeromelaban** [reply]
The test is running properly on our end. Make sure to follow [this documentation](https://platform.uno/docs/articles/external/uno.uitest/doc/using-uno-uitest.html#running-the-tests-for-webassembly) for getting started with UI Tests.
